packages/block-editor/src/components/block-tools/insertion-point.js

```diff
@@ -60,13 +64,17 @@ function InsertionPointPopover( {
 			_nextClientId = getNextBlockClientId( _nextClientId );
 		}
 
+		const settings = getSettings();
+
 		return {
 			previousClientId: _previousClientId,
 			nextClientId: _nextClientId,
 			orientation:
 				getBlockListSettings( insertionPoint.rootClientId )
 					?.orientation || 'vertical',
 			rootClientId: insertionPoint.rootClientId,
+			isNavigationMode: _isNavigationMode(),
+			hasReducedUI: settings.hasReducedUI,
```

Comment:
Can we update all the terms to be `distractionFree`? It's going to be harder to read the code if we differ longer term

Reply:
I wasn't sure we'd be dropping the other one. But sure yes! :)

Reply:
Done in [6276ad0](https://github.com/WordPress/gutenberg/pull/41740/commits/6276ad015aff2d498600293dbaf9bf5480d6c9bc)